docs/quick-start.md

```diff
@@ -14,20 +14,27 @@ For quick testing, you can use a local cluster with:
 * [k3s](https://k3s.io/) or [k3d](https://k3d.io/)
 * [Docker Desktop](https://www.docker.com/products/docker-desktop/)
 
+## Install Argo Workflows
+
+To install Argo Workflows for development or evaluation, use the
+`quick-start-minimal.yaml` manifest provided with each release.
+
 !!! Warning "Development vs. Production"
     These instructions are intended to help you get started quickly. They are not suitable for production.
     For production installs, please refer to [the installation documentation](installation.md).
 
-## Install Argo Workflows
+First, specify the version you'd like to install in an environment variable. Modify the
+command below:
 
-To install Argo Workflows, go to the [releases page](https://github.com/argoproj/argo-workflows/releases/latest).
-Scroll down to the `Controller and Server` section and execute the `kubectl` commands.
+```bash
+ARGO_WORKFLOWS_VERSION="vX.Y.Z"
```

Comment:
I was really hoping there was a way for Mkdocs to use the version number it "knows" about (from the upper-right corner), but failed to. I'm a newer Mkdocs user so it could have just been not knowing the right thing to search for, but I've done this before in Sphinx and couldn't figure it out in the time I had :upside_down_face:

Reply:
I'm pretty sure this requires a different plugin; `mkdocs-macros` [can do it](https://mkdocs-macros-plugin.readthedocs.io/en/latest/git_info/#tag-and-short_tag).

Reply:
Oo thank you :)